### ISSUE TITLE
val_loss of new model tacotron2 tts for persian language don't decreased

### ISSUE THREAD

**Issue by Lanzik** [NVIDIA/NeMo#5132]
I trained new model tacotron2 ( TTS ) with new data for persian language
but my val_loss is not normally

![Screenshot from 2022-10-11 01-36-20](https://user-images.githubusercontent.com/21003446/194959489-d45cd934-01be-415f-8743-55aaa74fc035.png)

I continued this training till 300 epochs but my val_loss was not normally decreased and it is constantly fluctuating

sample of one data in train or validation json file:
`{"audio_filepath": "./wav/4.wav", "duration": 5.6750625, "is_phoneme": 1, "text": "gAme sevvOm d%r b/rnamerizI  $enAxte moqe@iyy%te fe@lIye hozE @%st  ", "normalized_text": "gAme sevvOm d%r b/rnamerizI  $enAxte moqe@iyy%te fe@lIye hozE @%st  "}`

text is phoneticized but with uncommon method!
data is for persian(Farsi) language and I didn't use persian alphabet and I don't know what is my problem!
the output of my model is unclear!
loss diagram is normally but val_loss no
